docs/authentication/session-management-overview.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Session Management overview
+id: session-management-overview
+description: ''
+slug: /session-management-overview
+keywords:
+  - api token
+  - access token
+  - session management
+pagination_next: null
+pagination_prev: null
+last_update:
+  date: 08/08/2023
+  author: William May
+doc_type: overview
+displayed_sidebar: mainSidebar
+---
+
+import MultiLanguageCodeBlock from '@site/src/components/CodeBlocks/MultiLanguageCodeBlock';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## What is Session Management
```

Comment:
This might be useful to link to or get some information from https://cheatsheetseries.owasp.org/cheatsheets/Session_Management_Cheat_Sheet.html

Reply:
Thanks 👍